### DISCUSSION TITLE
Idea: Import settings from borgmatics config.yaml

### DISCUSSION THREAD

**Discussion by derphilipp** [borgbase/vorta#800] [Ideas]
It would be very nice for people migrating from borgmatic to be able to import (the config.yaml format)[https://github.com/witten/borgmatic].

This could be done by an additional "import from borgmatic" button on the UI, parsing the config.yaml and importing its settings into the options.

This is absolutely nice-to-have and probably not many people use borgmatic on Desktops anyway;

**Comment by m3nu** [comment]
Other users asked for an import/export feature for profiles. Which is needed eventually. It could use the same field names and sections as Borgmatic. And ignore the rest. Which would make it compatible to Borgmatic configs.

**Comment by ThomasWaldmann** [comment]
Interesting idea. For the overlap of features, vorta could be even a gui to make borgmatic confs maybe? :)

**Comment by witten** [comment]
Let me know if I can do anything on the borgmatic side to make the configuration easier to consume by other software! For instance, I could potentially do all the fancy include file resolution and output a single YAML file, a single JSON file, etc.

**Comment by m3nu** [comment]
Your YAML config is already quite well-structured. And I admit some concepts in Vorta are inspired by options in it. Like the list of sources, exclusions, etc. So Borgmatic configs map to Vorta options to a degree (also because they do the same thing). I wonder if Vorta could show Borgmatic files as read-only profiles. (as suggested above)

Too many ideas and too little time.

**Comment by witten** [comment]
A bunch of the concepts in borgmatic config are inspired by the Borg command-line, so we're all standing on the shoulders of giants, so to speak. :smile: 

Read-only profiles in Vorta would be cool. I know what you mean by too little time..

**Comment by samuel-w** [comment]
Borgmatic has config options that Vorta does not have, such as patterns. How should those be handled, as the only way I see is to ignore them.

**Comment by witten** [comment]
Seems reasonable to ignore them for now, maybe along with a warning for the user.
